src/components/TeamsLayout.tsx

```diff
@@ -29,26 +28,59 @@ const TeamsLayout: FunctionComponent = (props) => {
+                    <path d="M106.908596,16.4807453 C106.823548,15.6320934 106.532598,14.9465328 105.989491,14.5149905 C105.489653,14.0719019 104.780928,13.8308732 103.905092,13.8308732 C100.735971,13.8308732 100.274927,16.4865184 100.274927,19.0223709 C100.274927,21.5567802 100.735971,24.1691268 103.905092,24.1691268 C105.697048,24.1691268 106.948881,22.7604802 107.115991,21.1136915 L115,21.1136915 C114.206228,26.8074526 109.577879,30 103.737982,30 C97.2236782,30 92,25.6383914 92,19.0165978 C92,12.4020206 97.2236782,8 103.737982,8 C109.409277,8 114.249497,10.7061602 114.749335,16.4807453 L106.908596,16.4807453"></path>
+                    <polyline points="115 1 123.245818 1 123.245818 15.0160369 128.483831 8.93525108 137.794201 8.93525108 130.104448 16.8806756 139 30 129.106802 30 124.7014 22.3510574 123.245818 23.8959607 123.245818 30 115 30 115 1"></polyline>
+                    <path
+                      d="M139,8.55177515 L146.79367,8.55177515 L146.79367,11.2677515 L146.873792,11.2677515 C148.375712,9.1183432 150.528804,8 153.736591,8 C157.531453,8 161,10.3579882 161,15.285503 L161,30 L152.920805,30 L152.920805,18.7559172 C152.920805,16.2781065 152.63528,14.5414201 150.284068,14.5414201 C148.904516,14.5414201 147.077738,15.2426036 147.077738,18.6745562 L147.077738,30 L139,30 L139,8.55177515"
+                      id="Fill-18"
+                    ></path>
+                    <path d="M185,20.7649334 C185,12.6088781 181.402905,8 173.076217,8 C166.446107,8 162,12.9305619 162,19.0338994 C162,26.0618976 167.055025,30 173.763237,30 C178.531883,30 182.938938,27.9097764 184.557414,23.411973 L177.076846,23.411973 C176.436109,24.4130877 175.020123,24.9064324 173.727078,24.9064324 C171.219092,24.9064324 169.845051,23.1811684 169.641114,20.7649334 L185,20.7649334 Z M169.684505,16.621992 C170.046095,14.3485673 171.420136,13.0906826 173.802289,13.0906826 C175.864797,13.0906826 177.318388,14.6731362 177.318388,16.621992 L169.684505,16.621992 L169.684505,16.621992 Z"></path>
+                    <path d="M200.955311,28.5221077 C200.298901,30.3958684 199.56337,32.7989483 198.29304,34.3912803 C196.164103,37.0407925 192.887912,37 189.733333,37 L186.048352,37 L186.048352,30.5577263 L187.92967,30.5577263 C188.750183,30.5577263 189.813919,30.6408425 190.389744,30.3550393 C190.880586,30.1115234 191.1663,29.7032331 191.1663,28.7670818 C191.1663,27.7463561 188.054212,19.964052 187.604396,18.7406393 L184,9 L192.681319,9 L196.082051,21.1451774 L196.164103,21.1451774 L199.604396,9 L208,9 L200.955311,28.5221077"></path>
+                  </g>
```

Comment:
Could we extract all of this to its own component? I see it's in `ResidentLayout` as well?